src/pages/SignIn/SignIn.jsx

```diff
@@ -1,8 +1,21 @@
+import'./SignIn.css'
+
 function SignIn() {
     return (
-        <>
-            <h2>Sign in here</h2>
-        </>
+        <div className="background">
+            <div className="form-container">
+                <h2>Log In</h2>
+                <form>
```

Comment:
Klopt het dat je nog een submit functie mist? Als in, het daadwerkelijk gaan inloggen.

Reply:
ja dat klopt denk ik. zal er zo gelijk naar kijken